### PR TITLE
 [NL|FR|DE|IT|PT|ES Number] Fixed Relative Ordinal not complete across languages (#2827)

### PR DESCRIPTION
**Pull request by aitelint**
Fix to issue #2827 in NL|FR|DE|IT|PT|ES. 
Relative ordinals seem to be already supported in AR|HI|JA|KO|SV|TR.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs [hunk 1]
```diff
@@ -43,11 +43,12 @@ public static class NumbersDefinitions
       public static readonly string AllIntRegexWithLocks = $@"((?<=\b){AllIntRegex}(?=\b))";
       public static readonly string AllIntRegexWithDozenSuffixLocks = $@"(?<=\b)(((ein\s+)?halbes\s+dutzend)|({AllIntRegex}\s+dutzend(e)?))(?=\b)";
       public const string RoundNumberOrdinalRegex = @"(hundertst(er|es|en|el|e)?|tausendst(er|es|en|el|e)?|millionst(er|es|en|el|e)?|milliardst(er|es|en|el|e)?|billionst(er|es|en|el|e)?)";
-      public const string RelativeOrdinalRegex = @"(?<relativeOrdinal>(ante)?penultimate|letzter|nächster|vorheriger|aktueller)";
+      public const string RelativeOrdinalRegex = @"(?<relativeOrdinal>letzte[nr]?|nächste[nr]?|vorherige[nr]?|aktuelle[nr]?|jetzige[nr]?|vorletzte[nr]?|zuletzt|früher)";
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs:46]: Shouldn't we also cover drittletzter?

**Review by tellarin — CHANGES_REQUESTED** [review]
Thanks a lot! It would also be nice to have at least one spec per language in OrdinalModelSuppressExtendedTypes.json to show the flag works correctly.

File: .NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs (hunk 1 above)
Comment (tellarin) [reply .NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs:46]: vorletzte[nr]? and letzte[nr]? can also be combined in a simplified regex.

Comment (aitelint) [reply .NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs]: added drittletzter and simplified regex

**Comment by aitelint** [comment]
> Thanks a lot! It would also be nice to have at least one spec per language in OrdinalModelSuppressExtendedTypes.json to show the flag works correctly.

Added specs to OrdinalModelSuppressExtendedTypes.json

**Comment by tellarin** [comment]
@aitelint Build is failing as suppress cases are not disabled for JS. Also, please take a look at the issue as it seems there's something missing in the other languages too. Thanks.

**Comment by aitelint** [comment]
> @aitelint Build is failing as suppress cases are not disabled for JS. Also, please take a look at the issue as it seems there's something missing in the other languages too. Thanks.

yes, I will fix that soon.
I had a closer look at the other languages and apart for ZH that has no implementation of relative ordinals yet, there are a few specs missing in TR and SV (I will add them). 
The specs in the remaining languages (AR|HI|JA|KO) seem to cover all the patterns.
@nawanas, did you notice something missing in these languages?

**Review by tellarin — APPROVED** [review]

**Pull request merged**